### DISCUSSION TITLE
Emotes, icons and Emoji!

### DISCUSSION THREAD

**Discussion by FCLC** [hpc-social/mastodon-instance-and-policies#4] [Ideas]
Hi Everyone! 

I'll be sharing some sticker, icon and logo packs here to be used for the mast.hpc.social emoji keyboard! 

Feel free to suggest others as well! 

-Felix

https://emoji.gg/pack/7212-programming-emojis

https://commons.wikimedia.org/wiki/Category:Logos_of_Linux_distributions

https://commons.wikimedia.org/wiki/Category:Programming_language_logos

https://en.wikipedia.org/wiki/List_of_computing_mascots

**Comment by FCLC** [comment]
Currently missing from the above are: 
HPC guru (and various funny iterations like a "who is/I am/unmask HPC guru")
Some of the accelerator langs (MPI, OpenCL, Kokkos etc.)
Some of the "Memes", Xeon Phi, Itanium, Opteron/Bulldozer, Exascale 
I'd like to have some pride/outreach emojis/emotes (The ones based on the blob pack from android circa 2014-2017 are popular for this sort of application, BLAHAJ is another that's very low unassuming but popular)
VIM/EMACS should maybe make an appearance? 

I feel as if the HPC crowd might enjoy a Futurama reference or two with a zoidberg/professor emote? (Technically correct comes to mind?)

**Reply by alansill** [reply]
How can these be made available to account holders for use? Are there any restrictions involved, or is it just a technical issue of sharing?

**Reply by FCLC** [reply]
Once added to the instance, my understanding that they auto insert into the keyboard whenever you're logged in. 

Be it in browser or the various clients

**Reply by alansill** [reply]
Thanks. If you could find a link somewhere that we can provide to the hosting service, I can see if they can turn it on. Also I'd like to enable translation since we do sometimes get posts in other languages and I'd like to encourage that. I can read many at some level, but not all.

**Reply by alansill** [reply]
@FCLC Hereʻs a post on FediTips that confirms we can get these added. Do you have a package I should submit to get them added? I found the page to do this in the administrative interface. https://mstdn.social/@feditips/109604461015803758
